Coffee machines for all Galactic Conquest maps & Syndicate coffee mood

someone543 opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request
Adds coffee machines to all PVP maps and alters the mood event depending on the drinkers faction
<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->

## Why It's Good For The Game
The Syndicate shouldn't want to blow themselves up whenever they drink coffee.


<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding.
-->

## Testing Photographs and Procedure
<!-- Include any screenshots/videos/debugging steps of the modified code functioning successfully, ideally including edge cases. -->
<details>
<summary>Screenshots&Videos</summary>
Dark Astraeus

![SPACEcoffee](https://github.com/BeeStation/NSV13/assets/95106800/e8280f94-4c27-4e6d-970c-f342126d8c71)

Babylon

![BabyCoffee](https://github.com/BeeStation/NSV13/assets/95106800/4cd8725d-7547-463f-ae9a-9a9d8ef24244)

![BabyCoffee3](https://github.com/BeeStation/NSV13/assets/95106800/ab00d01e-0974-4378-affa-08819f329520)


Hammurabi


![HammurabiCoffee](https://github.com/BeeStation/NSV13/assets/95106800/606043b0-d4e3-4ad0-973c-fcc7d48b9167)

Mood event

![SyndiCoffee](https://github.com/BeeStation/NSV13/assets/95106800/8e381560-2c70-4cbc-9ba1-9c3aef61253b)

</details>

## Changelog
:cl:
add: Added coffee machines to PVP ships
tweak: Syndicate coffee drinkers now get a altered mood event
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

Bobbanz1 commented:
I now realized that I never thought about the Syndicate getting their hands on naval coffee back when I added the mood event for this stuff, also no premium coffee makers for the syndies?

Bokkiewokkie reviewed (changes requested):
Looks like you accidentally put some coffee in the southern Babylon cafeteria wall

someone543 commented:
> Looks like you accidentally put some coffee in the southern Babylon cafeteria wall

Oops. Fixed.


> I now realized that I never thought about the Syndicate getting their hands on naval coffee back when I added the mood event for this stuff, also no premium coffee makers for the syndies?

Unacceptable. Fixed, the Babylon now exclusively has it in the bridge.
